src/Maestro/Maestro.Data/Models/DependencyFlow.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
some of these files have the license and some don't

Reply:
I only added 1 file manually, the others were auto-generated, so they didn't get the license (and seem to not tend to).